src/core/js/views/componentView.js

```diff
@@ -5,6 +5,14 @@ define([
 
     var ComponentView = AdaptView.extend({
 
+        attributes: function() {
+            return {
+                "data-adapt-id": this.model.get('_id'),
+                "aria-labelledby": this.model.get('_id')+"-heading",
+                "role": "region"
+            };
+        },
```

Comment:
Would like to see this extending the results of the parent's [function](https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/views/adaptView.js#L7-L11) for future-proofing.

_Note: fix results helper to pass through correct context._